test/orchestratord/mzcompose.py

```diff
@@ -1930,10 +1934,12 @@ def measure_downtime() -> None:
 
     test_failed = False
     max_downtime = 15
-    for downtime in downtimes:
-        if downtime > max_downtime:
-            print(f"SELECT 1 took more than {max_downtime}s: {downtime}s")
-            test_failed = True
+    upgrade_downtime = downtimes[-1]
```

Comment:
I'm not sure why the initial downtime became higher. Maybe we made some change that is related. I'll try to figure it out.

Reply:
Seems kind of sporadic, so hard to pin down to some change :/